### PR TITLE
Bump 2 modules for D11

### PR DESCRIPTION
**Pull request by weitzman**
See bot comment for changed packages

**Jira:** (Skip unless you are MA staff)
DP-35789

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Changelog is not valid the following item(s) needs to be fixed (data should NOT have additional properties)</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5349153be6683164dba4fc83eab175e9e1b352fb
</p>

File: composer.lock [hunk 1]
```diff
@@ -2829,26 +2829,26 @@
         },
         {
             "name": "drupal/asset_cache_bust",
-            "version": "1.0.4",
+            "version": "1.0.5",
```
[the remark this answers is not in the document]
Comment (weitzman) [reply composer.lock:2832]: Bot you are drunk. This is not a problem.

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 97964d43237468b01cda03c4eadd1fba2a980168
</p>

**Comment by tom-fleming** [comment]
LGTM

**Pull request merged**